src/main/java/com/example/gamelist/GameMapper.java

```diff
@@ -2,12 +2,23 @@
 
 import org.apache.ibatis.annotations.Mapper;
 import org.apache.ibatis.annotations.Select;
+import org.apache.ibatis.annotations.Update;
 
 import java.util.List;
+import java.util.Optional;
+
 
 @Mapper
 public interface GameMapper {
 
+    //データを一括で取得する。　GET
     @Select("SELECT * FROM gamelists")
     List<Game> findAll();
+
+    @Select("SELECT * FROM gamelists WHERE id = #{id}")
+    Optional<Game> findById(int id);
+
+    @Update("UPDATE gamelist SET name = {#name} WHERE id = #{id}")
```

Comment:
テーブル名は gamelist 、 gamelists どっちでしょうか？
```suggestion
    @Update("UPDATE gamelists SET name = {#name} WHERE id = #{id}")
```

Reply:
fe99cbbdfcdc7ae6314a8a316eda7a3313cda81a

すみません、gamelist『s』にしないといけませんでした。
確認不足でした。申し訳ありません。

Reply:
OKです。